encoding/graph/thing/mod.rs

```diff
@@ -22,23 +23,35 @@ pub mod vertex_object;
 
 const THING_VERTEX_LENGTH_PREFIX_TYPE: usize = PrefixID::LENGTH + TypeID::LENGTH;
 
-pub trait ThingVertex<'a>: Prefixed<'a, BUFFER_KEY_INLINE> + Typed<'a, BUFFER_KEY_INLINE> {
+const fn max(lhs: usize, rhs: usize) -> usize {
+    if lhs < rhs {
+        rhs
+    } else {
+        lhs
+    }
+}
+
+pub const THING_VERTEX_MAX_LENGTH: usize = max(ObjectVertex::LENGTH, AttributeVertex::MAX_LENGTH);
```

Comment:
Standard `max` is not `const` (comes from trait `Ord`)

Reply:
nice.